platform-api/basic-authentication-guide.md

```diff
@@ -0,0 +1,154 @@
+# Basic Authentication Guide
```

Comment:
@cpk1224 I feel like this might need a little more context to help the user. I think you're really just teaching the user how to use basic auth to do some testing, but I think it should be made clear this is not how you would do it if you have a need to call APIs in a production app.